tests/queries/0_stateless/02902_parseDateTime_hour_no_leading_zero.sql

```diff
@@ -0,0 +1,14 @@
+-- { echo }
+
+-- %h
+select parseDateTime('Aug 13, 2022, 7:58:32 PM', '%b %e, %G, %h:%i:%s %p', 'UTC'); -- { serverError CANNOT_PARSE_DATETIME }
```

Comment:
Let's adjust/extend the existing test 02668_parse_datetime.sql instead.

Reply:
Ok, updated. (AFAIR the policy was different).